src/components/MultiSelect/index.d.ts

```diff
@@ -0,0 +1,27 @@
+import { ReactNode, FocusEvent } from 'react';
+import { BaseProps } from '../types';
+
+export interface MultiSelectOption {
+    name?: string;
+    label?: string;
+}
+
+export interface MultiSelectProps extends BaseProps {
+    label?: ReactNode;
+    placeholder?: string;
+    error?: ReactNode;
+    bottomHelpText?: ReactNode;
+    tabIndex?: string | number;
+    required?: boolean;
+    disabled?: boolean;
+    readOnly?: boolean;
+    variant?: 'default' | 'bare';
+    chipVariant?: 'base' | 'neutral' | 'outline-brand' | 'brand';
+    hideLabel?: boolean;
+    value?: MultiSelectOption | Array<MultiSelectOption>;
```

Comment:
in this component, we will use an array with strings as the value and for the onChange like:
```
value?: string[];
onChange?: (value: string[]) => void;
```
where the string is the name that identifies the option